docs/topics/cache.txt

```diff
@@ -123,6 +123,27 @@ When using the ``pylibmc`` binding, do not include the ``unix:/`` prefix::
         }
     }
 
+You can set CLIENT_OPTIONS in OPTIONS setting to pass additional parameter
```

Comment:
use double backticks around code stuff like CLIENT_OPTIONS for proper formatting.